docs/source/guides/mod-config.md

```diff
@@ -3,55 +3,161 @@
 
 Mods can create new GUIs. One GUI that that MWSE ships with is the Mod Configuration Menu. By registering into this system, mods can create settings pages for their mods in a place where users can easily find and manipulate them.
 
+## Simplest Example
 
-!!! warning "Out of Date Information"
-    The information here is out of date, and does not use the new EasyMCM system. These docs will be updated in the future to cover EasyMCM features. In the meantime, consider using [EasyMCM's documentation](https://easymcm.readthedocs.io/en/latest/).
-
```

Comment:
I'd prefer if you left a link to EasyMCM's docs but with a different text, such as:
```md
!!! note
    The complete MCM API documentation doesn't exist yet. You can browse its predecessor [EasyMCM's documentation](https://easymcm.readthedocs.io/en/latest/).
```